packages/psyche/psyche.0.0.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "0918nobita"
+synopsis: "A WASM-friendly lightweight programming language implemented in OCaml"
+authors: [
+  "0918nobita"
+]
+homepage: "https://github.com/0918nobita/psyche"
+dev-repo: "git+https://github.com/0918nobita/psyche.git"
+bug-reports: "https://github.com/0918nobita/psyche/issues"
+build: [
+  [make]
+]
+depends: [
+  "ocaml" {>= "4.04.1" & < "4.06.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.04.1"}
```
Any reasons why it has such tight constraints? I just tried to compile it with OCaml 4.07 and 4.08 and it compiles perfectly.

Reply:
Originally I was going to use `ucorelib`, but I didn't use it.
So, from now on,  there is no special reason.
I'll fix the constraints.